.github/ISSUE_TEMPLATE/stable-bug.yml

```diff
@@ -0,0 +1,107 @@
+name: "\U0001F41E Stable branch bug report"
+description: Submit a bug report to help us improve the stable version of Iroha on "iroha2" branch
```

Comment:
```suggestion
description: Submit a bug report to help us improve the stable version of Iroha ([`iroha2`](https://github.com/hyperledger/iroha/tree/iroha2) branch)
```

Reply:
Same as above, with link rendering being broken, we can't use this yet:
![](https://i.imgur.com/moLs8it.png)